src/test/regress/expected/multi_metadata_sync.out

```diff
@@ -101,8 +101,8 @@ HINT:  Connect to worker nodes directly to manually change schemas of affected o
 SELECT unnest(master_metadata_snapshot());
                                                                                                                                                                                                                                                                                                                                                                                                          unnest                                                                                                                                                                                                                                                                                                                                                                                                         
 ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
- TRUNCATE pg_dist_node
  SELECT worker_drop_distributed_table(logicalrelid) FROM pg_dist_partition
+ TRUNCATE pg_dist_node
```

Comment:
I should try it again now that I've pushed back resolving the shard placement, but it used to fail because the placements didn't have associated nodes. Actually it probably still will, because we'll want to read out the node the placements are on in order to delete them.